asm/non_matchings/audio/external/func_800C76C0.s

```diff
@@ -473,8 +473,8 @@ glabel func_800C76C0
 /* 0C89A4 800C7DA4 93C40043 */   lbu   $a0, 0x43($fp)
 /* 0C89A8 800C7DA8 0C0323BE */  jal   func_800C8EF8
 /* 0C89AC 800C7DAC 24040017 */   li    $a0, 23
-/* 0C89B0 800C7DB0 3C19800E */  lui   $t9, %hi(D_800DC5E8) # $t9, 0x800e
-/* 0C89B4 800C7DB4 8F39C5E8 */  lw    $t9, %lo(D_800DC5E8)($t9)
+/* 0C89B0 800C7DB0 3C19800E */  lui   $t9, %hi(gWinningPlayerIndex) # $t9, 0x800e
```

Comment:
`gPlayerWinningIndex`?

Reply:
It's the index of the "winning player", but I guess your version makes sense too 👍